src/Security/Domain/Authentication/Model/LocalProvider.php

```diff
@@ -0,0 +1,243 @@
+            $credentials['login'],
+            $credentials['password'],
+            0,
+            $this->dependencyInjector['configuration_db'],
+            $log,
+            1,
+            "",
+            "WEB"
+        );
+        $this->debug(
+            '[LOCAL PROVIDER] local provider trying to authenticate using legacy Authentication',
+            [
+                "class" => \CentreonAuth::class,
+            ],
+            function () use ($auth) {
+                $userInfos = $auth->userInfos;
+                return [
+                    'contact_id' => $userInfos['contact_id'],
+                    'contact_alias' => $userInfos['contact_alias'],
+                    'contact_auth_type' => $userInfos['contact_auth_type'],
+                    'contact_ldap_dn' => $userInfos['contact_ldap_dn']
+                ];
+            }
+        );
+        if ($auth->passwdOk === 1) {
+            if ($auth->userInfos !== null) {
+                $this->contactId = (int) $auth->userInfos['contact_id'];
+                $this->setLegacySession(new \Centreon($auth->userInfos));
+            }
+            $this->isAuthenticated = true;
+            $this->info('[LOCAL PROVIDER] authentication succeed from legacy Authentication');
+        } else {
+            $this->isAuthenticated = false;
+        }
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getLegacySession(): ?\Centreon
+    {
+        return $this->legacySession;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function setLegacySession(?\Centreon $legacySession): void
+    {
+        $this->legacySession = $legacySession;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function canCreateUser(): bool
+    {
+        return false;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function refreshToken(AuthenticationTokens $authenticationTokens): ?AuthenticationTokens
+    {
+        return null;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getName(): string
+    {
+        return self::NAME;
+    }
+
+    public function getUser(): ?ContactInterface
+    {
+        return $this->contactService->findContact($this->contactId);
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getConfiguration(): ProviderConfiguration
+    {
+        return $this->configuration;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function setConfiguration(ProviderConfiguration $configuration): void
+    {
+        $this->configuration = $configuration;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function canRefreshToken(): bool
+    {
+        return false;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function isAuthenticated(): bool
+    {
+        return $this->isAuthenticated;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getProviderToken(string $token): ProviderToken
+    {
+        $expirationSessionDelay = 120;
```

Comment:
Expiration delay must be define in the Symfony configuration
```suggestion
        $expirationSessionDelay = 120;
```